assets/javascripts/discourse/initializers/custom-wizard-edits.js.es6

```diff
@@ -87,10 +87,13 @@ export default {
       api.modifyClass("component:category-chooser", {
         categoriesByScope(options = {}) {
           let categories = this._super(options);
-
-          return categories.filter((category) => {
-            return !category.custom_fields?.create_topic_wizard;
-          });
+          const currentUser = this.currentUser;
+          if (!(currentUser && (currentUser.admin || currentUser.staff))) {
```

Comment:
@jumagura This conditional can be shortened to
```
!currentUser?.staff
```
Note that staff includes admins.

Reply:
Thans for pointing it out. I've implemented the change